src/TinyPubSub/TinyPubSub.cs

```diff
@@ -334,6 +346,7 @@ public static TinyEventArgs PublishControlled<T>(string channel, T instance, Act
                     if (subscription.Owner.Target == null)
```

Comment:
Why doesn't just

subscriber.Owner?.Target == null 

be enough? Instead of creating a static object? Just trying to understand.